Nos/Views/KnownFollowersView.swift

```diff
@@ -31,36 +31,42 @@ struct KnownFollowersView: View {
         knownFollowers.prefix(3).map { $0.source?.profilePhotoURL }
     }
     
-    var followText: LocalizedStringKey {
+    var followText: Text {
+        let stringResource: LocalizedStringResource
         switch avatarURLs.count {
         case 1:
             guard let name = knownFollowers[safe: 0]?.source?.safeName else {
                 fallthrough
             }
-            return LocalizedStringKey(String(localized: LocalizedStringResource.localizable.followedByOne(name)))
+            stringResource = LocalizedStringResource.localizable.followedByOne(name)
         case 2:
             guard let firstName = knownFollowers[safe: 0]?.source?.safeName,
                 let secondName = knownFollowers[safe: 1]?.source?.safeName else {
                 fallthrough
             }
-            return LocalizedStringKey(
-                String(localized: LocalizedStringResource.localizable.followedByTwo(firstName, secondName))
-            )
+            stringResource = LocalizedStringResource.localizable.followedByTwo(firstName, secondName)
         case 3:
             guard let firstName = knownFollowers[safe: 0]?.source?.safeName,
                 let secondName = knownFollowers[safe: 1]?.source?.safeName else {
                 fallthrough
             }
-            return LocalizedStringKey(
-                String(
-                    localized: LocalizedStringResource.localizable.followedByTwoAndMore(
-                        firstName, secondName, followers.count - 2
-                    )
-                )
+            stringResource = LocalizedStringResource.localizable.followedByTwoAndMore(
+                firstName, secondName, followers.count - 2
             )
         default:
-            return ""
+            return Text("")
         }
+
+        let attributedString = AttributedString(localized: stringResource)
+            .replacingAttributes(
+                AttributeContainer(
+                    [.inlinePresentationIntent: InlinePresentationIntent.stronglyEmphasized.rawValue]
+                ),
+                with: AttributeContainer(
+                    [.foregroundColor: UIColor(.primaryTxt)]
```

Comment:
In Figma I'm seeing this text as bold and purple. Is this a bug? https://www.figma.com/file/s0qf4VmyQygydP8MIQazZc/Nos?type=design&node-id=5732-1749&mode=design&t=qeCUEQECL1XBvTO4-0

Reply:
@mplorentz Figma is outdated now. I had originally asked to do it bold & purple since I didn't know Martin had figured out how to do it in purple & white. So @Chardot  switched to bold & purple, but his original design was purple & white.